content/doc/book/security/index.adoc

```diff
@@ -9,30 +9,75 @@ ifdef::backend-html5[]
 :sectanchors:
 endif::[]
 
-Jenkins is used everywhere from workstations on corporate intranets, to high-powered servers connected to the public internet.
-To safely support this wide spread of security and threat profiles, Jenkins offers many configuration options for enabling, customizing, or disabling various security features.
-
-Many of the security options are enabled by default when passing the interactive setup wizard to ensure that Jenkins is secure.
-Others involve environment-specific setup and trade-offs and depend on specific use cases supported in individual Jenkins instances.
-
-This chapter will introduce the various security options available to Jenkins administrators and users, explaining the protections offered, and trade-offs to disabling some of them.
+Jenkins is used everywhere -- from workstations on corporate intranets to high-powered servers connected to the public internet.
+It is critically important to keep your Jenkins instance secure,both to protect your information and to avoid executing malicious code from your Jenkins instance.
+
+Your Jenkins environment is a fully-distributed build system.
+Each network connection is a potential point of entry.
+Remember that the code that runs your builds can be perverted to run anything!
+
+Jenkins and the jobs it runs must be able to do almost anything.
+This means that a malicious Pipeline could reconfigure the Jenkins instance, delete files, or launch various forms of mischief such as a DDoS attack or a bot.
+In addition to deliberate and direct attacks on your environment, a trusted user could visit an infected web site and accidentally introduce malicious code into the Jenkins instance.
```

Comment:
Second paragraph seems to repeat what the first one says?

Reply:
Did a little rewrite, which is still not perfect.